help/link-to-a-message-or-conversation.md

```diff
@@ -35,7 +35,7 @@ You can create a channel link manually by typing `#**channel name**`
 
 ## Link to a topic within Zulip
 
-Topic links are automatically formatted as [#channel > topic]().
+Topic links are automatically formatted as `[#channel > topic]()`.
```

Comment:
We're trying to show what they look like here, so I think a different change is required. Making the link just be to this page itself is potentially fine.